app/src/main/java/com/poupa/vinylmusicplayer/util/PreferenceUtil.java

```diff
@@ -557,6 +558,10 @@ public final boolean animatePlayingSongIcon() {
         return mPreferences.getBoolean(ANIMATE_PLAYING_SONG_ICON, false);
     }
 
+    public final boolean showSongNumber() {
+        return mPreferences.getBoolean(SHOW_SONG_NUMBER, false);
```

Comment:
So this means that the song number would be hidden by default? Please change to default to true so the current behaviour is unchanged

Reply:
No it is not the song number that the app already has, it is this one:
![91655755-7323e800-eab3-11ea-8b33-39f4a3a50038](https://user-images.githubusercontent.com/56130419/94951082-f6d25980-04e3-11eb-9cd1-86cacd5c1f6c.png)

Perhaps my naming scheme is not suggestive enough. But by default this MR doesn't change anything

Reply:
makes sense. Sorry I thought it was about hiding the number elsewhere